data/themes/default/hypha.css

```diff
@@ -740,3 +740,100 @@ th{
 body.type_peer_reviewed_article .input-wrapper.field_type_editor:not(.field_name_text) .wym_iframe iframe {
 	height: 150px;
 }
+
+.notelo {
+    visibility: visible;
+    background-color: #555;
+    color: #fff;
+    text-align: center;
+    border-radius: 6px;
+    position: absolute;
+    margin-left: -60px;
+    opacity: 1;
+    z-index: 12;
+}
+.notelo.up::after {
+    content: "";
+    position: absolute;
+    bottom: 100%;
+    left: 50%;
+    margin-left: -5px;
+    border-left: 10px solid transparent;
+    border-right: 10px solid transparent;
+    border-bottom: 10px solid #555;
+}
+.notelo.down::after {
+    content: "";
+    position: absolute;
+    top: 100%;
+    left: 50%;
+    margin-left: -5px;
+
+    border-left: 10px solid transparent;
+    border-right: 10px solid transparent;
+    border-top: 10px solid #555;
+}
+.notelo.left::after {
+    content: "";
+    position: absolute;
+    top: 50%;
+    left: 0%;
+    margin-top: -10px;
+    margin-left: -10px;
+
+    border-top: 10px solid transparent;
+    border-bottom: 10px solid transparent;
+    border-right: 10px solid #555;
+}
+.notelo.right::after {
+    content: "";
+    position: absolute;
+    top: 50%;
+    left: 100%;
+    margin-top: -10px;
+    border-top: 10px solid transparent;
+    border-bottom: 10px solid transparent;
+    border-left: 10px solid #555;
+}
+.closebutton {
+    position: absolute;
+    top: -10px;
+    background-color: #ff0000;
+    z-index: 15;
+}
+.closebutton:hover {
+    background-color: #ffffff;
+}
+.closebutton:focus {
+    background-color: #ffffff;
+    top: -5px;
+}
+.hyphaInfoButton {
+    background-color: #00f;
+    border: 2px solid #00f;
+    color: #FFF;
+    cursor: pointer;
+    display: inline-block;
+    font-family: arial;
+    font-weight: bold;
+    top: -15px;
+    right: -15px;
+    font-size: 15px;
+    line-height: 15px;
+    width: 15px;
+    height: 15px;
+    margin-left: 3px;
+    margin-right: 3px;
+    border-radius: 50%;
+    text-align: center;
+}
+.hyphaInfoButton:hover {
+    background-color: #fff;
+    color: #00f;
+}
+.hyphaInfoButton.clicked {
+    background-color: yellow;
+}
+.hyphaInfoButton:before {
+    content: 'i';
+}
```

Comment:
There is no newline at the end of this file. Git prefers to have each line terminated, which helps when generating diffs. Ok to leave like this, but perhaps your editor can be configured to insert these newlines?